pep/app/pods/components/modal-dialogs/help/report-data-error/component.ts

```diff
@@ -0,0 +1,126 @@
+import NotificationService from 'ember-cli-notifications/services/notifications';
+import IntlService from 'ember-intl/services/intl';
+import ENV from 'pep/config/environment';
+import { AUTHOR_INDEX_SUPPORT_URL, CUSTOMER_SERVICE_URL, SUGGEST_NEW_CONTENT_URL } from 'pep/constants/urls';
+import AjaxService from 'pep/services/ajax';
+import CurrentUserService from 'pep/services/current-user';
+import LoadingBarService from 'pep/services/loading-bar';
+import REPORT_DATA_ERROR_VALIDATIONS from 'pep/validations/help/report-data-error';
+
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import createChangeset, { ModelChangeset } from '@gavant/ember-validations/utilities/create-changeset';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface ErrorReport {
+    username: string;
+    fullName: string;
+    email: string;
+    problemText: string;
+    correctedText: string;
+    urlProblemPage: string;
+    additionalInfo: string;
+    isAuthorPublisher: boolean;
+    hasOriginalCopy: boolean;
+}
+
+interface SupportResource {
+    label: string;
+    href: string;
+}
+
+type ErrorReportChangeset = ModelChangeset<ErrorReport>;
+
+interface ModalDialogsHelpReportDataErrorArgs {
+    onClose: () => void;
+}
+
+export default class ModalDialogsHelpReportDataError extends Component<ModalDialogsHelpReportDataErrorArgs> {
+    @service ajax!: AjaxService;
+    @service currentUser!: CurrentUserService;
+    @service loadingBar!: LoadingBarService;
+    @service notifications!: NotificationService;
+    @service intl!: IntlService;
+    validations = REPORT_DATA_ERROR_VALIDATIONS;
+    dataErrorUrl = `${ENV.reportsBaseUrl}/data-errors`;
+
+    supportResources: SupportResource[] = [
```

Comment:
We usually separate stuff like this out into a constants file

Reply:
@bakerac4 yeah the reason I didn't put these entire objects in a constants file is because we already have a `url` constants file that these objects use and I didn't want to necessarily tie a specific "label" to the url (so you could more readily re-use the urls w/ different link labels, etc)

Reply:
@napafundi I think that you can do both as long as you just name things appropriately. So if you were to create an export called `SUPPORT_RESOURCES` in the `urls.ts` file and export what you have currently you could still not tie the labels in.

Also I just noticed this now, but these need to be translated. Check out `themes.ts` for how we normally do that.

Reply:
@bakerac4 ahh yep, good catch. Yeah I see, I was thinking of making a whole new file but this makes more sense.